compatible/backup_cluster.yaml

```diff
@@ -0,0 +1,204 @@
+      - ../bin:/go/src/github.com/pingcap/br/bin
+      - ./prepare_data:/prepare_data
+    command: -c "/usr/bin/tail -f /dev/null"
+    environment:
+      TAG: ${TAG}
+    depends_on:
+      - "tidb"
+    restart: on-failure
+    env_file:
+      - ../docker/minio.env
+
+  pd0:
+    image: pingcap/pd:${TAG}
+    ports:
+      - 237${PORT_SUFFIX}:2379
+    volumes:
+      - ../docker/config/pd.toml:/pd.toml:ro
+      - /tmp/br/docker/backup_data/${TAG}:/data
+      - /tmp/br/docker/backup_logs/${TAG}:/logs
+    command:
+      - --name=pd0
+      - --client-urls=http://0.0.0.0:2379
+      - --peer-urls=http://0.0.0.0:2380
+      - --advertise-client-urls=http://pd0:2379
+      - --advertise-peer-urls=http://pd0:2380
+      - --initial-cluster=pd0=http://pd0:2380
+      - --data-dir=/data/pd0
+      - --config=/pd.toml
+      - --log-file=/logs/pd0.log
+    # sysctls:
+    #   net.core.somaxconn: 32768
+    # ulimits:
+    #   nofile:
+    #     soft: 1000000
+    #     hard: 1000000
+    restart: on-failure
+
+  tikv0:
+    image: pingcap/tikv:${TAG}
+    volumes:
+      - ../docker/config/tikv.toml:/tikv.toml:ro
+      - /tmp/br/docker/backup_data/${TAG}:/data
+      - /tmp/br/docker/backup_logs/${TAG}:/logs
+    command:
+      - --addr=0.0.0.0:20160
+      - --advertise-addr=tikv0:20160
+      - --data-dir=/data/tikv0
+      - --pd=pd0:2379
+      - --config=/tikv.toml
+      - --log-file=/logs/tikv0.log
+    depends_on:
+      - "pd0"
+    # sysctls:
+    #   net.core.somaxconn: 32768
+    # ulimits:
+    #   nofile:
+    #     soft: 1000000
+    #     hard: 1000000
+    restart: on-failure
+    env_file:
+      - ../docker/minio.env
+
+  tikv1:
+    image: pingcap/tikv:${TAG}
+    volumes:
+      - ../docker/config/tikv.toml:/tikv.toml:ro
+      - /tmp/br/docker/backup_data/${TAG}:/data
+      - /tmp/br/docker/backup_logs/${TAG}:/logs
+    command:
+      - --addr=0.0.0.0:20160
+      - --advertise-addr=tikv1:20160
+      - --data-dir=/data/tikv1
+      - --pd=pd0:2379
+      - --config=/tikv.toml
+      - --log-file=/logs/tikv1.log
+    depends_on:
+      - "pd0"
+    # sysctls:
+    #   net.core.somaxconn: 32768
+    # ulimits:
+    #   nofile:
+    #     soft: 1000000
+    #     hard: 1000000
+    restart: on-failure
+    env_file:
+      - ../docker/minio.env
+
+  tikv2:
+    image: pingcap/tikv:${TAG}
+    volumes:
+      - ../docker/config/tikv.toml:/tikv.toml:ro
+      - /tmp/br/docker/backup_data/${TAG}:/data
+      - /tmp/br/docker/backup_logs/${TAG}:/logs
+    command:
+      - --addr=0.0.0.0:20160
+      - --advertise-addr=tikv2:20160
+      - --data-dir=/data/tikv2
+      - --pd=pd0:2379
+      - --config=/tikv.toml
+      - --log-file=/logs/tikv2.log
+    depends_on:
+      - "pd0"
+    # sysctls:
+    #   net.core.somaxconn: 32768
+    # ulimits:
+    #   nofile:
+    #     soft: 1000000
+    #     hard: 1000000
+    restart: on-failure
+    env_file:
+      - ../docker/minio.env
+
+  tikv3:
```

Comment:
If this cluster is only for generating backup archives, maybe one kv would be enough?

Reply:
yes, one kv is enough.